doc/modules/linear_model.rst

```diff
@@ -1111,17 +1124,18 @@ The minimization problem becomes:
 where :math:`\alpha` is the L2 regularization penalty. When sample weights are
 provided, the average becomes a weighted average.
 
-The following table lists some specific EDMs and their unit deviance (all of
-these are instances of the Tweedie family):
+The following table lists some specific EDMs and their unit deviance :
 
-================= ===============================  ============================================
+================= ================================  ============================================
 Distribution       Target Domain                    Unit Deviance :math:`d(y, \hat{y})`
-================= ===============================  ============================================
-Normal            :math:`y \in (-\infty, \infty)`  :math:`(y-\hat{y})^2`
-Poisson           :math:`y \in [0, \infty)`        :math:`2(y\log\frac{y}{\hat{y}}-y+\hat{y})`
-Gamma             :math:`y \in (0, \infty)`        :math:`2(\log\frac{\hat{y}}{y}+\frac{y}{\hat{y}}-1)`
-Inverse Gaussian  :math:`y \in (0, \infty)`        :math:`\frac{(y-\hat{y})^2}{y\hat{y}^2}`
-================= ===============================  ============================================
+================= ================================  ============================================
+Normal            :math:`y \in (-\infty, \infty)`   :math:`(y-\hat{y})^2`
+Bernoulli         :math:`y \in \{0, 1\}`            :math:`2({y}\log\frac{y}{\hat{y}}+({1}-{y})\log\frac{{1}-{y}}{{1}-\hat{y}})`
+Categorical       :math:`y \in \{0, 1, ..., k-1\}`  :math:`2\sum_{i=0}^{k_\text{classes}-1} y_\text{i}\log\frac{y_\text{i}}{\hat{y_\text{i}}}`
```

Comment:
In this notation, `y_i` is an (ordinal class label) integer, so it does not work. Think you need to use a binary class indicator variable.